scripts/fanshow

```diff
@@ -11,13 +11,15 @@ from swsssdk import SonicV2Connector
 from natsort import natsorted
 
 
-header = ['FAN', 'Speed', 'Direction', 'Presence', 'Status', 'Timestamp']
+header = ['Drawer', 'FAN', 'Speed', 'Direction', 'Presence', 'Status', 'LED', 'Timestamp']
```

Comment:
Move 'LED' to right of 'Drawer'

Reply:
In this case you probably need to update the commit message 
Please add previous and new output for comparison

Reply:
Updated.